packages/sfui/frameworks/vue/components/SfRatingButton/SfRatingButton.vue

```diff
@@ -72,6 +72,8 @@ const handleHoverOut = () => {
 };
 </script>
 
+<!-- eslint-disable vuejs-accessibility/mouse-events-have-key-events -->
```

Comment:
I think we can disable this rule whatsover - in almost every case we handle somehow keyboard, but sometime even do this via refs (and composables). I worry that this rule will just fail too often

Reply:
I was thinking about this, will disable it globally

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2915/commits/bfe2a3bed414a6223f8f9e50a7e95e3f2728d351